pkg/README.md

```diff
@@ -40,8 +40,8 @@ find_system_rpmbuild(name = "rules_pkg_rpmbuild")
 This example is a simplification of the debian packaging of Bazel:
 
 ```python
-load("@rules_pkg//pkg:pkg.bzl", "pkg_tar", "pkg_deb")
-
+load("@rules_pkg//pkg:pkg.bzl", "pkg_tar")
+load("@rules_pkg//pkg:pkg_deb.bzl", "pkg_deb")
```

Comment:
I wonder if this file could just be be called "deb.bzl" instead of "pkg_deb.bzl".

IMO, it's a bit redundant to have `pkg` said twice in the label.  As an example of prior art,`rpm.bzl` and `rpm_pfg.bzl` contain `pkg_rpm`.

Reply:
Good point. I'll rename.